static/styles/compose.css

```diff
@@ -478,6 +478,11 @@ textarea.new_message_textarea {
         cursor: not-allowed;
         background-color: hsl(0, 0%, 93%);
     }
+
+    &.invalid,
+    &.invalid:focus {
```

Comment:
I guess we need the `:focus` here to override the base `:focus` color?

Reply:
Yup exactly.